actions/windows.js

```diff
@@ -78,11 +79,20 @@ export function raiseWindow(id) {
     };
 }
 
-export function setSplitScreen(windowId, side, size) {
+export function setSplitScreen(windowId, side, size, menuMargins) {
     return {
         type: SET_SPLIT_SCREEN,
         windowId: windowId,
         side: side,
-        size: size
+        size: size,
+        menuMargins: menuMargins
```

Comment:
I don't think there is any advantage in passing the `menuMargins` here, since it is identical to the state value. Just use `menuMargins` in the reducer. You can move the `const mapMargins = {...}` block in the reducer to a separate funtion which you call both as a result of `SET_SPLIT_SCREEN` and `SET_MENU_MARGIN`.

Reply:
I'm sorry I don't understand how to move the `const mapMargins = {...}` block to a separate function. mapMargin Values in  `SET_SPLIT_SCREEN` and `SET_MENU_MARGIN` are different...